src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/cjsm/CjsmService.kt

```diff
@@ -48,6 +48,8 @@ class CjsmService(
           }
       }
 
+  fun findOrganisation(secureEmail: String): String? = cjsmDirectoryRepository.findBySecureEmail(secureEmail)?.organisation
```

Comment:
Would an optional rather than a nullable make the consuming code easier to write and subsequently grok maybe?